draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -154,6 +155,18 @@ document are to be interpreted as described in {{RFC2119}}.
 Description {#description}
 ======================
 
+This document extends the {{ICESDP}} specification to allow the generation
+and resolution of ICE candidates that use mDNS FQDN names.
```

Comment:
We don't have a definiton of what an mDNS FQDN name is. How about "a specific subset of FQDN addresses"?  "FQDN addresses" is used instead of "FQDN names" to be consistent with ICESDP and to also avoid the repetition of  "name".